test/fragment-cases/trailing-linebreaks.html

```diff
@@ -1 +1 @@
-trailing linebreaks are allowed
+trailing linebreaks are allowed
```

Comment:
Here the input test file has three line breaks at the end. The first two start a new paragraph. The last is consumed as paragraphs consume any leading whitespace.